src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1553,8 +1546,8 @@ private void NotifyFeaturesAboutValueChange(DatabaseRecord record, long index, s
                         continue;
 
                     DatabaseShutdown.ThrowIfCancellationRequested();
-                    SubscriptionStorage?.HandleDatabaseRecordChange(record);
-                    EtlLoader?.HandleDatabaseValueChanged(record);
+                    SubscriptionStorage?.HandleDatabaseRecordChange(record.Topology);
```

Comment:
only here you actually need the database record, so let's open it here?
and avoid a situation that you have many opened read tx